### PR TITLE
1/n : fiat support - model changes

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- added modal changes needed to support fiat 
- added help text for bounty fulfillments
- added custom migration to update existing data for newly added fields 
- added TODO comments to alter fields name for later 

Fields added:

| Field                | Description                                                                              |
|----------------------|------------------------------------------------------------------------------------------|
| payout_type          | fiat / qr / standard_bounties                                                            |
| tenant               | each payout_type has it's unique set on tenants Eg: celo for qr bounties / pypl for fiat |
| funder_address       | same as bounty_owner_address but will be updated later on for shared mgmt                |
| funder_profile       | same as bounty_owner_profile but will be updated later on for shared mgmt                |
| funder_identifier    | unique funder identifier in fiat flow (like pypl username)                               |
| fulfiller_identifier | unique fulfiller identifier for when (like pypl username)                                |

##### Testing

- std bounties flow (creation -> fulfillment -> payout)
- cross chain flow (creation -> fulfillment -> payout)
- ran custom migration to udpate exsisting db 

Screenshot 2020-05-16 at 7 55 37 PM
Screenshot 2020-05-16 at 7 55 44 PM

======

Screenshot 2020-05-17 at 9 59 34 AM

Screenshot 2020-05-17 at 9 59 28 AM

### CODE REVIEW

File: app/dashboard/models.py [hunk 1]
```diff
@@ -1333,25 +1333,53 @@ class BountyFulfillment(SuperModel):
...
-    accepted = models.BooleanField(default=False)
-    accepted_on = models.DateTimeField(null=True, blank=True)
 
-    bounty = models.ForeignKey(Bounty, related_name='fulfillments', on_delete=models.CASCADE)
-    profile = models.ForeignKey('dashboard.Profile', related_name='fulfilled', on_delete=models.CASCADE, null=True)
+    accepted = models.BooleanField(default=False, help_text="has the fulfillment been accepted by the funder")
+    accepted_on = models.DateTimeField(null=True, blank=True, help_text="date when the fulfillment was accepted by the funder")
+
+    payout_type = models.CharField(max_length=20, null=True, blank=True,choices=PAYOUT_TYPE, help_text="payment type used to make the payment")
+    tenant = models.CharField(max_length=10, null=True, blank=True, help_text="specific tenant type under the payout_type")
```
Comment (danlipert) [inline app/dashboard/models.py:1374]: choices here as well? what do you think?

File: app/dashboard/migrations/0110_auto_20200517_0514.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.4 on 2020-05-16 13:19
+
+from django.db import migrations, models
+
+def populateFunderAddress(apps, schema_editor):
+    BountyFulfillment = apps.get_model('dashboard', 'BountyFulfillment')
+
+    for fulfillment in BountyFulfillment.objects.all():
+        fulfillment.funder_address = fulfillment.bounty.bounty_owner_address
```
Comment (danlipert) [inline app/dashboard/migrations/0110_auto_20200517_0514.py:9]: nice! Can we backfill some of the other fields as well?

Comment (thelostone-mc) [reply app/dashboard/migrations/0110_auto_20200517_0514.py:9]: like ? I'm not sure which one else we would need to backfill -> 
I've got a follow up PR which retires a few fields which don't have data 

@danlipert

File: app/dashboard/models.py (hunk 1 above)
Comment (thelostone-mc) [reply app/dashboard/models.py:1374]: tenant would be super long list cause of the standard bounties 
For every ERC-20 token -> tenant == token_name // This is along list
For cross chain -> tenant == token_name except cUSD / cGLD
For fiat -> tenant = PYPL

File: app/dashboard/migrations/0110_auto_20200517_0514.py (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/migrations/0110_auto_20200517_0514.py:9]: I'll have that PR opened up today ! Just testing out flow

**Pull request merged**